frontend/src/pages/Domains/index.tsx

```diff
@@ -3,7 +3,7 @@ import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
 import { Typography } from "@material-ui/core";
 import Nav from "src/components/Nav";
 import DomainCard from "src/components/Cards/DomainCard";
-import { teamList } from "./teamList";
+import { teamList_22, teamList_21, teamList_20 } from "./teamList";
```

Comment:
Instead of defining multiple lists for batches, can we have a field in the user object that specifies the batch?
That will be much more clear and easy to maintain.

Reply:
I have done the changes as requested. Kindly review it and let me know if any further adjustment is required